Cargo.toml

```diff
@@ -31,26 +31,22 @@ spanish = []
 default = ["chinese-simplified", "chinese-traditional", "french", "italian", "japanese", "korean", "spanish"]
 
 [dependencies]
-anyhow = "1.0.34"
-thiserror = "1.0.22"
-rustc-hash = "1.1.0"
-sha2 = "0.9.1"
-hmac = "0.8.1"
-pbkdf2 = { version = "0.4.0", default-features = false }
-rand = "0.7.3"
-once_cell = { version = "1.8.0" }
-unicode-normalization = "0.1.13"
-zeroize = { version = "1.1.1", features = ["zeroize_derive"] }
+anyhow = "1.0"
```

Comment:
I can understand dropping patch versions from for example `rand` that was upgraded to the next minor version. But why do you allow in more buggy/vulnerable versions of those crates that were not upgraded? Could you please add those patch versions back, where you have not upgraded at least a minor version? Thanks!

(Switching between Rust and TypeScript I was always confused that `1.0.34` translates to `^1.0.34` that matches anything `>=1.0.34,<2` unlike `npm`, where `1.0.34` would translate to `=1.0.34`)

Reply:
Reverted. I didn't realize Cargo's versioning worked like that. My mental model was closer to `npm`'s. Thanks for the tip!